ang/test/mocks/data/cases-types.data.js

```diff
@@ -328,16 +328,28 @@
 
   CRM['civicase-base'].caseTypes = _.clone(caseTypesMock);
 
-  module.service('CaseTypes', function () {
-    return {
-      /**
-       * Returns a list of case types
-       *
-       * @returns {Array} case types
-       */
-      get: function () {
-        return _.clone(caseTypesMock);
-      }
+  module.provider('CaseTypesMockData', function () {
```

Comment:
Why do we define this as a Provider instead of a service?

Reply:
if we add it as a provider we can add more case types to it in other extensions, specifically awards:

https://github.com/compucorp/uk.co.compucorp.civiawards/pull/19/files#diff-02e65478f14b1b21fd0835d8f4479255R4-R8

If we don't add it as a provider then we'd need to manually add the case type values from award in this extension. Civicase should not know about any particular extension.